src/tests/store/activities/actions.test.js

```diff
@@ -0,0 +1,39 @@
+import { addActivity, removeActivity, editActivity } from '../../../store/activities/actions';
+import { MONDAY } from '../../../constants/dates';
+
+test('should setup add activity action object', () => {
```

Comment:
you can use `describe` block to describe what are you testing. 
better word will be `it` instead of `test`, then sentence will be created: `it should....` :)